plugin/possession.lua

```diff
@@ -1,4 +1,12 @@
 local group = vim.api.nvim_create_augroup('Possession', {})
+local nvim_received_stdin = false
```

Comment:
I don't think we need to use a vim global, a module variable seems to work for all we need here.

Reply:
Right, that's nice